conanfile.py

```diff
@@ -17,6 +17,8 @@ class CrashpadConan(ConanFile):
     url = "https://github.com/bincrafters/conan-crashpad"
     topics = ("conan", "crash-reporting", "logging", "minidump", "crash")
     settings = "os", "compiler", "build_type", "arch"
+    options = {'linktime_optimization': [True, False]}
+    default_options = "linktime_optimization=True"
```

Comment:
Given that conan cannot properly deal with the link time optimization flag: Should we disable this by default?

Reply:
Yes I was thinking about this as well. But hopefully conan will fix in the near future?